modules/s3/s3codecs/pdf.py

```diff
@@ -90,6 +91,17 @@
 PDF_WIDTH = 0
 PDF_HEIGHT = 1
 
+if current.response.s3.language == "tr":
```

Comment:
Hmm... so a dict which would store the lang code as key and the font used by that language as the values ?